django/db/backends/postgresql/psycopg_any.py

```diff
@@ -0,0 +1,47 @@
+import ipaddress
+
+try:
+    from psycopg import ClientCursor, IsolationLevel, errors, sql
+    from psycopg.types.json import Jsonb
+    from psycopg.types.range import Range
+
+    Inet = ipaddress.ip_address
+
+    DateRange = DateTimeRange = DateTimeTZRange = NumericRange = Range
+    RANGE_TYPES = (Range,)
+
+    def mogrify(sql, params, connection):
+        return ClientCursor(connection.connection).mogrify(sql, params)
+
+    is_psycopg3 = True
+
+except ImportError:
+    from enum import IntEnum
+
+    from psycopg2 import errors, extensions, sql  # NOQA
+    from psycopg2.extras import DateRange, DateTimeRange, DateTimeTZRange, Inet  # NOQA
+    from psycopg2.extras import Json as Jsonb  # NOQA
+    from psycopg2.extras import NumericRange, Range  # NOQA
+
+    def _quote(value, cursor_or_connection=None):
```

Comment:
[`psycopg`'s `sql.quote()`](https://www.psycopg.org/psycopg3/docs/api/sql.html#psycopg.sql.quote) accepts `sql` and `context` so we should rename the second argument.